server/mod.ts

```diff
@@ -1,3 +1,3 @@
-export * from "./cache/mod.ts"
-export * from "./local.ts"
-export * from "./server.ts"
+export * from "./Env.ts"
```

Comment:
I couldn't comment on the favicon file, so: thoughts on the favicon being changed to the logo I made?

Reply:
YES –– wanna swap it out with yours?